roles/manage-sshd-config/tasks/sshd-update.yml

```diff
@@ -0,0 +1,10 @@
+---
+- name: "Update sshd_config with Dict" 
```

Comment:
I'd just change it to "Update sshd_config with provided values" (or something like that). I think the user may find "with Dict" to be confusing